ivy/functional/ivy/random.py

```diff
@@ -536,12 +536,17 @@ def shuffle(
     ret
         An array object, shuffled along the first dimension.
 
-    Examples
-    --------
+    Functional Examples
+    -------------------
     >>> x = ivy.array([1, 2, 3, 4, 5])
     >>> y = ivy.shuffle(x)
     >>> print(y)
     ivy.array([2, 1, 4, 3, 5])
 
+    >>> x = ivy.array([1,0,5])
+    >>> y = ivy.array(ivy.zeros((3,)))
```

Comment:
please use change this to `ivy.array([ 0, 0, 0 ])` as it is more clear than using a function call 😅